infra/scripts/bash-helpers.sh

```diff
@@ -91,6 +91,8 @@ addr-from-dockerhost () {
 #    export COMPUTE=cluster/pool
 #    export DATASTORE=datastore1
 #    export IMAGE_STORE=$DATASTORE/image/path
+#    export MAPPED_NETWORKS="--container-network=VM Network:external"
```

Comment:
You need to add an example of how to supply multiples - they need to be an array or the expansion won't work correctly for CLI processing:
```
MAPPED_NETWORKS=("--container-network=VM Network:external" "--container-network=SomeOtherNet:elsewhere")
export MAPPED_NETWORKS
```